app/controllers/katello/api/v2/content_export_incrementals_controller.rb

```diff
@@ -66,9 +66,17 @@ def find_export_view_helper(name_importable, name_syncable, organization, genera
     end
 
     def find_library_export_view
+      importable_name = ::Katello::ContentView::EXPORT_LIBRARY
+      syncable_name = "#{::Katello::ContentView::EXPORT_LIBRARY}-SYNCABLE"
+
+      unless params[:destination_server].blank?
+        importable_name += "-#{params[:destination_server]}"
+        syncable_name += "-#{params[:destination_server]}"
+      end
```

Comment:
Why aren't we using `::Katello::Pulp3::ContentViewVersion::Export.find_library_export_view` like we did before 4da582b3f7e81339ae1feded66a85758850ddfaa -- that does the `name += "-#{destination_server}" unless destination_server.blank?` part already?

Otherwise this obviously works, but seems like duplication?